src/generator.js

```diff
@@ -33,11 +41,35 @@ function createPHPFile(folder) {
             return
         }
 
+        const category = detectCategory(name);
+
+        if (category == 'class') {
+            return window.showQuickPick([
+                "Class",
+                "Trait",
+                "Interface"
+            ], {
+                title: "Choose class type",
+                canPickMany: false,
+                matchOnDescription: true,
+                matchOnDetail: true
+            });
+        }
```

Comment:
sorry, but I don't want a QuickPick for this, detectCategory already return "class", so let's use "class"

Most of the time it's what the user wants, so I'm not happy with a question with a QuickPick that's mostly useless

```suggestion
```

Reply:
or maybe with an option in package.json thats default to false ?